site/content/docs/user/quick-start.md

```diff
@@ -202,12 +202,26 @@ In particular, many users may be interested in multi-node clusters. A simple
 configuration for this can be achieved with the following config file contents:
 ```yaml
 # three node (two workers) cluster config
```

Comment:
this comment doesn't match now

Reply:
hm, it says 3 nodes total, with 2 workers.
```
# three node (two workers) cluster config
kind: Cluster
apiVersion: kind.sigs.k8s.io/v1alpha3
nodes:
- role: control-plane
- role: worker
- role: worker
```

should this be changed?

Reply:
Derp, thanks